src/content/docs/es/guides/rss.mdx

```diff
@@ -206,6 +206,31 @@ export function GET(context) {
 }
 ```
 
+## Removing trailing slashes
+
```

Comment:
Looks like you forgot to translate this heading!

Reply:
Nice catch I will fix this shortly